src/linear_algebra/quotient.lean

```diff
@@ -438,3 +438,40 @@ def mapq_linear : compatible_maps p q →ₗ[R] (M ⧸ p) →ₗ[R] (M₂ ⧸ q)
 end submodule
 
 end comm_ring
+
+section as_quotient_group
+
+namespace submodule
+
+/-- The group underlying the module `M ⧸ S` is isomorphic to the quotient group `M ⧸ S`. -/
+noncomputable def quotient_equiv_quotient_group {R M : Type*} [ring R] [add_comm_group M]
+  [module R M] (S : submodule R M) : M ⧸ S ≃+ M ⧸ S.to_add_subgroup :=
```

Comment:
Of course this would remove the problem, but I'm not sure I want to go in such a big refactor right now :sweat_smile: If anyone wants to, that is indeed a good idea (there's even a TODO about this above the definition of `submodule.quotient`). That said, I don't think the current situation is too terrible to work with, I'd rather have a few `equiv`s in some places rather than a lot of `rw` failures and having to `change` all the time.

Reply:
I've opened #13620 to try that.

Reply:
This is now merged, so either you can replace this with `add_equiv.refl`, or abandon the PR.

Reply:
Thanks ! I'm doing that right now

Reply:
Well actually I'll just close this PR and open another one with the two lemmas about kernels.